sklearn/metrics/tests/test_classification.py

```diff
@@ -1628,3 +1629,20 @@ def test_brier_score_loss():
     # calculate even if only single class in y_true (#6980)
     assert_almost_equal(brier_score_loss([0], [0.5]), 0.25)
     assert_almost_equal(brier_score_loss([1], [0.5]), 0.25)
+
+
+@if_numpydoc
+def test_docstring():
+    # Test for consistency among docstring of different metrics
+    assert_consistent_docs([precision_recall_fscore_support, precision_score,
+                            recall_score, f1_score, fbeta_score],
+                           include_returns=False,
+                           exclude_params=['labels', 'average', 'beta'])
+
+    error_str = ("Parameter 'labels' of 'precision_score' has inconsistent "
```

Comment:
I assume we don't want this inconsistency to exist? The docs should be fixed then.

Reply:
In `precision_score` description there seems to be an addition,
`.. versionchanged:: 0.17
           parameter *labels* improved for multiclass problem.`
Should this be added in `precision_recall_fscore_support`? If yes then would this PR be appropriate?

Reply:
I suppose so.